src/prefect/core/task.py

```diff
@@ -207,7 +207,9 @@ def run(self, x, y):
             and at write time all formatting kwargs will be passed and a fully formatted location is
             expected as the return value. The callable can be used for string formatting logic that
             `.format(**kwargs)` doesn't support. **Note**: this only works for tasks running against a
-            backend API.
```

Comment:
To document this, I'd only modify the sentence currently ending on line 205. Something like:

```diff
-            formatted at runtime with values from `prefect.context`, task inputs, and parameters.
+            formatted at runtime with values from task arguments, `prefect.context`, and flow
+            parameters (in the case of a name conflict between these, earlier values take precedence).
```

(unfortunately Github won't let me make a suggestion involving lines outside your diff)